src/main/java/net/dv8tion/jda/internal/entities/UserImpl.java

```diff
@@ -43,6 +44,8 @@
     protected long privateChannel = 0L;
     protected boolean bot;
     protected boolean fake = false;
+    protected EnumSet<UserFlag> flagsSet;
```

Comment:
This shouldn't be stored, just compute it when its needed.

Reply:
Was changed and is now computed in `getFlags()`